realm/src/androidTest/java/io/realm/RealmLinkTests.java

```diff
@@ -272,7 +273,8 @@ public void testQuerySingleRelationString() {
         RealmResults<Owner> owners5 = testRealm.where(Owner.class).contains("cat.name", "Blackie").findAll();
         assertEquals(1, owners5.size());
 
-        RealmResults<Owner> owners6 = testRealm.where(Owner.class).equalTo("cat.name", "blackIE", RealmQuery.CASE_INSENSITIVE).findAll();
+        RealmResults<Owner> owners6 = testRealm.where(Owner.class).equalTo("cat.name", "blackIE", RealmQuery
+                .CASE_INSENSITIVE).findAll();
```

Comment:
Yeah... Apparently AS really sucks at line wrapping. I will disable the wrappings there and simply enforce 120 in checkstyle letting freedom to the developer to break at the right place